Obvs.Serialization.MessagePack-CSharp/Obvs.Serialization.MessagePack-CSharp.csproj

```diff
@@ -10,8 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MessagePack" Version="1.7.3.2" />
-    <PackageReference Include="Obvs" Version="4.1.0.9" />
+    <PackageReference Include="MessagePack" Version="1.7.3" />
```

Comment:
Any reason for this downgrade?  I see `MessagePack 1.7.3.4` is out ? https://www.nuget.org/packages/MessagePack/